gnovm/tests/files/addressable_7a_err.gno

```diff
@@ -0,0 +1,12 @@
+package main
+
+func foo() ([]int, []string) {
+	return []int{1, 2, 3}, []string{"a", "b", "c"}
+}
+
+func main() {
+	_ = &foo()
```

Comment:
Do you think we can do the check in AssignStmt? I have similar concern while reviewing this PR: https://github.com/gnolang/gno/pull/2695

Reply:
resolve this cuz it should be fixed in another fix -- out of scope.